accounts/abi/abi.go

```diff
@@ -117,11 +118,66 @@ func (abi ABI) Pack(name string, args ...interface{}) ([]byte, error) {
 	return append(method.Id(), arguments...), nil
 }
 
+// toGoSliceType prses the input and casts it to the proper slice defined by the ABI
+// argument in T.
+func toGoSlice(i int, t Argument, output []byte) (interface{}, error) {
+	index := i * 32
+	if index+96 > len(output) {
+		return nil, fmt.Errorf("abi: cannot marshal in to go slice type: length insufficient %d require %d", len(output), index+64)
+	}
```

Comment:
Why `index+96`? And why in the log `index+64`? Shouldn't this only check index + 32 as that's what contains the offset. Which can point very far out, so we'll need to check that later?